packages/gauges/src/constants/config/prod.ts

```diff
@@ -5077,4 +5077,44 @@ export const CONFIG_PROD: GaugeConfig[] = [
     token1Address: ethereumTokens.weth.address,
     feeTier: FeeAmount.MEDIUM,
   },
+  {
+    gid: 513,
+    address: '0x9E143b6644C752d82F63a416c1f37c908f845BB4',
+    pairName: 'WIF-ETH',
+    chainId: ChainId.BASE,
+    type: GaugeType.V3,
+    token0Address: baseTokens.wif.address,
+    token1Address: baseTokens.weth.address,
+    feeTier: FeeAmount.MEDIUM,
+  },
+  {
+    gid: 514,
+    address: '0x62Fcb3C1794FB95BD8B1A97f6Ad5D8a7e4943a1e',
+    pairName: 'ETH-BNB',
+    chainId: ChainId.BSC,
+    type: GaugeType.V3,
+    token0Address: bscTokens.eth.address,
+    token1Address: bscTokens.wbnb.address,
+    feeTier: FeeAmount.LOWEST,
+  },
+  {
+    gid: 515,
+    address: '0x9F599F3D64a9D99eA21e68127Bb6CE99f893DA61',
+    pairName: 'ETH-USDT',
+    chainId: ChainId.BSC,
+    type: GaugeType.V3,
+    token0Address: bscTokens.eth.address,
+    token1Address: bscTokens.usdt.address,
+    feeTier: FeeAmount.LOWEST,
+  },
+  {
+    gid: 516,
+    address: '0x247f51881d1E3aE0f759AFB801413a6C948Ef442',
+    pairName: 'ETH-USDT',
```

Comment:
@ChefMomota Hi. The name doesn't match the tokens here

Reply:
done